app/web/.env.development

```diff
@@ -4,6 +4,4 @@ NEXT_PUBLIC_MEDIA_BASE_URL="https://dev-user-media.couchershq.org"
 NEXT_PUBLIC_CONSOLE_BASE_URL="https://next-console.couchershq.org"
 NEXT_PUBLIC_MAPBOX_KEY="pk.eyJ1IjoiY291Y2hlcnMiLCJhIjoiY2tpYzY4eHd2MGVpcTJ0bGdhdGhvbHhlbiJ9.u5zvDeFE9H8itTK_dNp7Pg"
 NEXT_PUBLIC_NOMINATIM_URL="https://nominatim.openstreetmap.org/"
-NEXT_PUBLIC_TILE_URL="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
-NEXT_PUBLIC_TILE_ATTRIBUTION="Map data &copy; <a href="https://osm.org/copyright">OpenStreetMap</a> contributors"
```

Comment:
Are these two being removed because we switched out which map provider we're using?

Reply:
Back in the day (a very long time ago) we used to embed an OSM raster base map (pngs). These are from those days. Nowadays we use mapbox which delivers vector base tiles. We’re moving to self hosted base tiles which is the next step.

Reply:
Ah gotcha so not gonna break anything, excellent!